.clang-format

```diff
@@ -108,7 +108,7 @@ SpacesInContainerLiterals: true
 SpacesInCStyleCastParentheses: false
 SpacesInParentheses: false
 SpacesInSquareBrackets: false
-Standard:        Cpp11
+Standard:        Latest
```

Comment:
Are you sure? Maybe it would be better to specify a known standard?

Reply:
Yes, I'm sure ) It's not mine change, I've got this file from upstream.

Reply:
I suppose, @nest1ing, they are controlling it with the version of clang-format binary they use in Yocto build.

Reply:
I've been confused by this change because it may bring unexpected behaviour with update `clang` in the future.
But, as Artem said it was brought by the upstream.
Probably, they know a little bit more about this. So, I think let it be as is.